tsdb/db.go

```diff
@@ -276,6 +282,10 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_size_retentions_total",
 		Help: "The number of times that blocks were deleted because the maximum number of bytes was exceeded.",
 	})
+	m.lockfileCreatedCleanly = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_clean_start",
+		Help: "0: a lockfile from a previous execution was replaced. 1: lockfile creation was clean. 2: lockfile is disabled",
```

Comment:
The code is doing -1 for "lockfile disabled" (which I like more than 2, actually; 2 would suggest "super clean", while -1 tell me "something's irregular").

Reply:
Forgot to update that 😬